doc/library/config.txt

```diff
@@ -528,6 +528,13 @@ import theano and print the config variable, as in:
     performance, check both options.
 
 
+.. attribute:: config.gpuarray.cache_path
+
+   String value
+
+   Directory to cache pre-compiled kernels for the gpuarray backend.
```

Comment:
Tell that it default to the subdirectory gpuarray_kernels inside theano compile dir.